reddit_liveupdate/templates/liveupdatemediaembedbody.html

```diff
@@ -45,7 +45,7 @@
         if ('MutationObserver' in window) {
             /* Currently just check on every set of mutations. */
             var observer = new MutationObserver(checkDimensions),
-                config = { attributes: true, subtree: true};
+                config = { attributes: true, subtree: true, childList: true};
```

Comment:
:no_bell: I'm surprised that `subtree` doesn't also take into account all direct children already - does this do something different?